fuzz/fuzz_targets/full_stack_target.rs

```diff
@@ -265,6 +265,10 @@ impl KeysInterface for KeyProvider {
 			}
 		}
 	}
+
+	fn get_session_key(&self) -> SecretKey {
+		SecretKey::from_slice(&Secp256k1::without_caps(), &[0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0]).unwrap()
```

Comment:
Oops, please avoid changing the keys used in things in the fuzz targets. You can just take https://github.com/TheBlueMatt/rust-lightning/commit/5a829777a67bd5d3c09ae4c5e6ef2cf81c88c6af, I think.